scripts/after-bump.ts

```diff
@@ -1,107 +1,26 @@
-import { resolve, relative } from 'path';
-import { formatJson, unixEOL } from '@sap-cloud-sdk/util';
-import { apiDocsDir, transformFile, currentSdkVersion } from './util';
-
-function updateRootPackageJson() {
-  transformFile(resolve('package.json'), packageJson =>
-    formatJson({
-      ...JSON.parse(packageJson),
-      version: `${currentSdkVersion}`
-    })
-  );
-}
-
-function updateDocumentationMd() {
-  transformFile(resolve('DOCUMENTATION.md'), documentation =>
-    documentation
-      .split(unixEOL)
-      .map(line =>
-        line.startsWith('## Version:')
-          ? `## Version: ${currentSdkVersion}`
-          : line
-      )
-      .join(unixEOL)
-  );
-}
-
-function updateTypeDocConfig() {
-  transformFile('tsconfig.typedoc.json', config => {
-    const parsedConfig = JSON.parse(config);
-    return formatJson({
-      ...parsedConfig,
-      typedocOptions: {
-        ...parsedConfig.typedocOptions,
-        out: `${relative(resolve(), apiDocsDir)}/${currentSdkVersion}`
-      }
-    });
-  });
+import { readFile } from 'fs/promises';
+import { add, commit, tag } from '@changesets/git';
+import { mergeChangelogs } from './merge-changelogs';
+import { generateDocs } from './generate-docs';
+
+async function getVersion() {
```

Comment:
Instead of reimplementing this, you could use the `currentSdkVersion` that already exists, right?